### PR TITLE
PDP: Update Team members content

### LINKED ISSUE #3206: PDP: Team members
Add docs for PDP Team management:

- Team members (concept)
	- Invite team members to a project (task)
	- Remove team members (task)
	- Change the roles of team members (task)
	- Roles and permissions (reference)

Size: S
Linear issue: https://linear.app/prisma-company/issue/CP-3100/docs-update-team-members-content

### PR DESCRIPTION
**Pull request by zachtil**
## Describe this PR

Update content in the Members section

## Changes

* Reorganize and rename files
* Rename "Members" to "Team members" (simultaneously being updated in PDP UI)
* Resolve conflicts on members, team members, collaborators terminology
* 
## What issue does this fix?

Fixes #3206
Fixes CDN-3100

### CODE REVIEW

File: content/800-data-platform/400-members/050-roles-permissions.mdx [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
-To add users, you need to use their GitHub username and they need to have an account in the Platform.
+To add users, you need to use their GitHub username and they need to have an account in the platform.
 
-</TopBlock>
+## Permissions
+
+When you assign a role to a team member, you control what they can view or edit in the project.
 
-## Roles and permissions
+The table below lists all permissions that different roles give or restrict.
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/050-roles-permissions.mdx]: I think the key thing here is that these are predefined sets of permissions available to each role. When you assign a role, it gives them that set of permissions.

Therefore, I think we should frame this in terms of which permissions the roles you give.

So I'd suggest something like this:

"Each role has a fixed set of permissions. When you assign a role to a team member, you grant them the set of permissions that applies to the role. The table below shows the permissions for each role."

File: content/800-data-platform/400-members/100-add.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: 'Add a team member'
+metaTitle: 'Add a team member'
+metaDescription: 'In Prisma Data Platform, you can invite team members to a project when you just define GitHub username and assign them a role.'
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/100-add.mdx]:
```suggestion
metaDescription: 'In Prisma Data Platform, you can invite team members to a project and assign them a role.'
```

I don't think we need the GitHub part (that's a "how to" detail). If you want to keep it however, there seems to be something missing from that part of the sentence. Should it say "**their** GitHub username"?

File: content/800-data-platform/400-members/100-add.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+---
+
+<TopBlock>
+
+In Prisma Data Platform, you can invite team members to a project when you just define GitHub username and assign them a role.
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/100-add.mdx]:
```suggestion
In Prisma Data Platform, you can invite team members to a project. To invite a team member, you provide their GitHub username and assign them a role.
```

A few things with this sentence:
- There appears to be missing a word here - do you mean "**their** GitHub username"?
- I think "define" is wrong here, because that suggests that you create the username in some way. Maybe "provide" or "specify"?
- I think we should avoid the "just". We have a style guideline against saying things are easy.

File: content/800-data-platform/400-members/100-add.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+</TopBlock>
+
+## Prerequisites
+
+- You are the [Project owner](/data-platform/members/roles-permissions) or have the [Admin](/data-platform/members/roles-permissions) role.
+- The new team member must have a GitHub and must have used that to log in to Prisma Data Platform.
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/100-add.mdx]:
```suggestion
- The new team member must log in to Prisma Data Platform with a GitHub account.
```

I think we can shorten this to the above.

File: content/800-data-platform/400-members/100-add.mdx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+- Obtain their GitHub username.
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/100-add.mdx]:
```suggestion
- You know the user's GitHub username.
```

File: content/800-data-platform/400-members/100-add.mdx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+- Check your [project plan](/data-platform/billing/plans-and-quotas) and see if you have quota to add new team members.
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/100-add.mdx]:
```suggestion
- Your [project plan](/data-platform/billing/plans-and-quotas) has available quota to add new team members.
```

File: content/800-data-platform/400-members/100-add.mdx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+<TopBlock>
+
+In Prisma Data Platform, you can invite team members to a project when you just define GitHub username and assign them a role.
+
+</TopBlock>
+
+## Prerequisites
+
+- You are the [Project owner](/data-platform/members/roles-permissions) or have the [Admin](/data-platform/members/roles-permissions) role.
```
Comment (tatianainama) [inline content/800-data-platform/400-members/100-add.mdx:15]: I see links from the `change-role` and `remove` pages are bold, but these are not, should we make it consistent?

File: content/800-data-platform/400-members/100-add.mdx [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+- Obtain their GitHub username.
+- Check your [project plan](/data-platform/billing/plans-and-quotas) and see if you have quota to add new team members.
+
+## Steps
+
+1. From the [Projects](https://cloud.prisma.io/projects) page, select the project to which you want to add a team member.
+2. From the **Projects** drop-down in the upper-left, select **Project settings**.
+3. Click **Team members**.
+4. Type the GitHub username in the box and click **Invite team member**.
+5. In the pop-up window, select a [role](/data-platform/members/roles-permissions) for the new team member from the **Role** drop-down menu.
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/100-add.mdx]: Do we need to mention the pop-up window?

Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/100-add.mdx]:
```suggestion
5. In the **Role** drop-down menu, select a [role](/data-platform/members/roles-permissions) for the new team member.
```

File: content/800-data-platform/400-members/100-add.mdx [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+    alt="Team members - invite and select a role"
+    width="475px"
+  />
+
+6. Click **Save**.
+
+## Result
+
+- Your new project collaborator can now use their GitHub account to log in to [https://cloud.prisma.io](https://cloud.prisma.io).
+- The collaborator can now open your project and edit data in **Data Browser**.
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/100-add.mdx]: This is more a style guide discussion point, rather than an issue with this doc, but to my ear **the** Data Browser sounds better.

File: content/800-data-platform/400-members/300-change-role.mdx [hunk 10]
```diff
@@ -0,0 +1,29 @@
+---
+title: 'Change the role of a team member'
+metaTitle: 'Change the role of a team member'
+metaDescription: 'You can change the role of any of the invited team members on a project.'
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/300-change-role.mdx]:
```suggestion
metaDescription: 'You can change the role of any of the project team members.'
```

Do we need to state "invited" here?

File: content/800-data-platform/400-members/400-remove.mdx [hunk 11]
```diff
@@ -0,0 +1,24 @@
+---
+title: 'Remove a team member'
+metaTitle: 'Remove a team member'
+metaDescription: 'You can remove any of the invited team members if they no longer participate or in case you want to reduce the team member overages on a project.'
+---
+
+<TopBlock>
+
+You can remove any of the invited team members if they no longer participate or in case you want to reduce the team member overages on a project.
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/400-remove.mdx]:
```suggestion
You can remove any of the invited team members, for example if they leave the team.
```

I'm not sure we should talk about reducing costs here

File: content/800-data-platform/400-members/400-remove.mdx [hunk 12]
```diff
@@ -0,0 +1,24 @@
+---
+title: 'Remove a team member'
+metaTitle: 'Remove a team member'
+metaDescription: 'You can remove any of the invited team members if they no longer participate or in case you want to reduce the team member overages on a project.'
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/400-remove.mdx]: See my comment below

File: content/800-data-platform/400-members/index.mdx [hunk 13]
```diff
@@ -1,12 +1,12 @@
...
-metaDescription: 'Add members to your project in the Prisma Data Platform to collaborate on administrating, developing, or editing the data of your project.'
+title: 'Team members'
+metaTitle: 'Team members'
+metaDescription: 'Add team members to your project in the Prisma Data Platform to collaborate on administrating, developing, or editing the data of your application that uses Prisma.'
 ---
 
 <TopBlock>
 
-Add members to your project in the Prisma Data Platform to collaborate on administrating, developing, or editing the data of your project.
+Add team members to your project in the Prisma Data Platform to collaborate on administrating, developing, or editing the data of your application that uses Prisma.
```
Comment (andrew-walford-prisma) [inline content/800-data-platform/400-members/index.mdx]: Maybe something like...

```suggestion
Add team members to your project in the Prisma Data Platform. Team members with appropriate roles can administrate and edit the data of your Prisma applications.
```

I simplified this somewhat. I wasn't sure what "develop" meant here, so I removed it. If you want to include it, then I suggest changing the order to:

```suggestion
Add team members to your project in the Prisma Data Platform. Team members with appropriate roles can edit the data, administrate, and develop your Prisma applications.
```

File: content/800-data-platform/400-members/100-add.mdx (hunk 7 above)
Comment (zachtil) [reply content/800-data-platform/400-members/100-add.mdx:15]: Thanks @tatianainama . I've now removed the bold from all of these so they are consistent.

**Review by andrew-walford-prisma — COMMENTED** [review]
I made a few suggestions - otherwise, it looks good!

File: content/800-data-platform/400-members/050-roles-permissions.mdx (hunk 1 above)
Comment (zachtil) [reply content/800-data-platform/400-members/050-roles-permissions.mdx]: Thank you for providing the wording. That works quite well!

File: content/800-data-platform/400-members/100-add.mdx (hunk 2 above)
Comment (zachtil) [reply content/800-data-platform/400-members/100-add.mdx]: The GitHub part is crucial because we do not have another mechanism yet (it is currently in development).

File: content/800-data-platform/400-members/100-add.mdx (hunk 3 above)
Comment (zachtil) [reply content/800-data-platform/400-members/100-add.mdx]: Yes, there were a number of issues here. I agree with all of them and have made the suggested change.

File: content/800-data-platform/400-members/100-add.mdx (hunk 4 above)
Comment (zachtil) [reply content/800-data-platform/400-members/100-add.mdx]: Do you think the use of present perfect here makes more sense? This is where present simple sounds more like a future action and, therefore, sounds like a "what to do next" for me.

My goal is to indicate that this is an action that should have already happened. I wasn't sure myself yesterday, but indeed, if you have a GitHub account but have not yet logged into the platform, you cannot be invited. Logging in with the GitHub account means that you now also have a platform account.

Comment (zachtil) [reply content/800-data-platform/400-members/100-add.mdx]: Changed this to: 

_The new team member must have logged in to the platform with a GitHub account._

File: content/800-data-platform/400-members/100-add.mdx (hunk 5 above)
Comment (zachtil) [reply content/800-data-platform/400-members/100-add.mdx]: Changed to the following to avoid possessive s:

_You know the GitHub username of the new team member._

File: content/800-data-platform/400-members/100-add.mdx (hunk 6 above)
Comment (zachtil) [reply content/800-data-platform/400-members/100-add.mdx]: Thank you. I like this suggestion. Changed now.

File: content/800-data-platform/400-members/100-add.mdx (hunk 8 above)
Comment (zachtil) [reply content/800-data-platform/400-members/100-add.mdx]: I find it useful but we can skip it.

Comment (zachtil) [reply content/800-data-platform/400-members/100-add.mdx]: From my previous training, we've always stressed that you select an option _from_ a drop-down. I've kept that. Otherwise, shortened as you suggest.

File: content/800-data-platform/400-members/100-add.mdx (hunk 9 above)
Comment (zachtil) [reply content/800-data-platform/400-members/100-add.mdx]: Let's discuss during the next docs weekly or during a style guide meeting.

I've had a big change on this last week from Lucy's review and would not want to do it again unless we review examples and decide on clear guidelines.

File: content/800-data-platform/400-members/300-change-role.mdx (hunk 10 above)
Comment (zachtil) [reply content/800-data-platform/400-members/300-change-role.mdx]: Definitely not. An oversight on my part. Thank you for flagging it.

File: content/800-data-platform/400-members/400-remove.mdx (hunk 11 above)
Comment (zachtil) [reply content/800-data-platform/400-members/400-remove.mdx]: I've removed that bit now.

Comment (zachtil) [reply content/800-data-platform/400-members/400-remove.mdx]: Changed to:

_You can remove a team member from a project when, for example, they leave the team._

File: content/800-data-platform/400-members/index.mdx (hunk 13 above)
Comment (zachtil) [reply content/800-data-platform/400-members/index.mdx]: Have we settled on using "Prisma application". I thought this was still quite debatable.

Otherwise, we have the Developer role which is for team members who are developers. The overall actions are indeed administrating, developing, editing data.

We also have the Viewer role which is only about viewing, but I don't think this is necessary to add here.

**Review by andrew-walford-prisma — APPROVED** [review]
